modules/ROOT/pages/clustering/disaster-recovery.adoc

```diff
@@ -3,21 +3,18 @@
 [[cluster-recovery]]
 = Disaster recovery
 
-Databases can become unavailable for different reasons.
-For the purpose of this section, an _unavailable database_ is defined as a database that is incapable of serving writes, while still may be able to serve reads.
-Databases not performing as expected for other reasons are not considered unavailable and cannot be helped by this section.
-//Refer to <<link to error handling section, TBD>> for more information on troubleshooting.
-This section contains a step-by-step guide on how to recover databases that have become unavailable.
-By performing the actions described here, the unavailable databases are recovered and made fully operational with as little impact as possible on the other databases in the cluster.
+A database can become unavailable due to issues on different system levels.
+For example, a data centre failover may lead to the loss of multiple servers, which may cause a set of databases to become unavailable.
+It is also possible for databases to become quarantined due to a critical failure in the system, which may lead to unavailability even without the loss of servers.
 
-There are many reasons why a database becomes unavailable and it can be caused by issues on different levels in the system.
-For example, a data-center failover may lead to the loss of multiple serves which in turn may cause a set of databases to become unavailable.
-It is also possible for databases to become quarantined due to a critical failure in the system which may lead to unavailability even without loss of servers.
+This section contains a step-by-step guide on how to recover databases that have become unavailable in terms of write operations, while still being able to serve read operations.
```

Comment:
I like the clarify of:
> an _unavailable database_ is defined as a database that is incapable of serving writes

Could this be worked into the new text?

Reply:
Thanks for the quick review, @jackwaudby. Which text do you refer to?

Reply:
Ah, I think I got it.